README.md

```diff
@@ -32,7 +34,7 @@ For the main LLVM repo, the Clang repo, and the CompilerRT repo, use the Memoro
 [CompilerRT](https://github.com/epfl-vlsc/compiler-rt)
 
 These repos should default to branch r40\_dev (Memoro is based off of LLVM release\_40).
-Optionally compile with libcxx and libcxxabi. 
+Optionally compile with libcxx and libcxxabi. If this does not work, use branch memoro_80.
```

Comment:
The branch memoro_80 is the active branch. We should direct people directly to memoro_80.

Reply:
I second this, r40_dev is pretty obsolete at this point.